mergin/client.py

```diff
@@ -70,6 +70,8 @@ def __init__(self, url=None, auth_token=None, login=None, password=None, plugin_
         self._auth_params = None
         self._auth_session = None
         self._user_info = None
+        self._server_type = None
+        self._global_namespace = None
```

Comment:
is this `_global_namespace` still needed?

Reply:
fixed